src/applications/disability-benefits/all-claims/tests/pages/evidenceTypes.unit.spec.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import { mount } from 'enzyme';
+import formConfig from '../../config/form';
+
+describe('evidenceTypes', () => {
+  const { schema, uiSchema } = formConfig.chapters.supportingEvidence.pages.evidenceTypes;
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{}}
+        formData={{}}/>
+    );
+
+    expect(form.find('input').length).to.equal(2);
+  });
+
+  it('should require selecting evidence or no evidence', () => {
```

Comment:
Ohh, I get it now. Took me a while. What do you think of "should require selecting an option?"

Reply:
Sometimes words are hard.